jenkins/internal-ci/generic/branch/motr.groovy

```diff
@@ -0,0 +1,296 @@
+                    sh label: '', script: '''
+                    yum-config-manager --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/rockylinux/rockylinux-8.4-2.0.0-latest/
+                    yum --nogpgcheck -y --disablerepo="EOS_Rocky_8_OS_x86_64_Rocky_8" install libfabric-1.11.2 libfabric-devel-1.11.2
+                    '''
+
+                    sh label: '', script: '''
+                        export build_number=${BUILD_ID}
+                        cp cortx-motr.spec.in cortx-motr.spec
+                        sed -i "/BuildRequires.*kernel*/d" cortx-motr.spec
+                        sed -i "/BuildRequires.*%{lustre_devel}/d" cortx-motr.spec
+                        sed -i 's/@BUILD_DEPEND_LIBFAB@//g' cortx-motr.spec
+                        sed -i 's/@.*@/111/g' cortx-motr.spec
+                        yum-builddep -y --nogpgcheck cortx-motr.spec
+                    ''' 
+                }
+            }
+        }
+
+        stage('Build') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                dir ('motr') {    
+                    sh label: '', script: '''
+                        rm -rf /root/rpmbuild/RPMS/x86_64/*.rpm
+                        ./autogen.sh
+                        ./configure --with-user-mode-only
+                        export build_number=${BUILD_ID}
+                        make rpms
+                    '''
+                }    
+            }
+        }
+        
+        stage ('Upload') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Copy RPMS', script: '''
+                    mkdir -p $build_upload_dir/$BUILD_NUMBER
+                    cp /root/rpmbuild/RPMS/x86_64/*.rpm $build_upload_dir/$BUILD_NUMBER
+                '''
+                sh label: 'Repo Creation', script: '''pushd $build_upload_dir/$BUILD_NUMBER
+                    rpm -qi createrepo || yum install -y createrepo
+                    createrepo .
+                    popd
+                '''
+            }
+        }
+        
+        stage ('Set Current Build') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Tag last_successful', script: '''
+                    pushd $build_upload_dir/
+                    test -d $build_upload_dir/current_build && rm -f current_build
+                    ln -s $build_upload_dir/$BUILD_NUMBER current_build
+                    popd
+                '''
+            }
+        }
+        
+        stage ("Trigger Downstream Jobs") {
+            parallel {
+                stage ("build S3Server") {
+                    when { expression { false } }
+                    steps {
+                        script { build_stage = env.STAGE_NAME }
+                        script {
+                            try {
+                                def s3Build = build job: 's3server', wait: true,
```

Comment:
do we need s3?

Reply:
This is disable at - https://github.com/shailesh-vaidya/cortx-re/blob/rocky-linux-custom-ci/jenkins/internal-ci/generic/branch/motr.groovy#L134 
Kept it for support in case we need to use RGW build details here. Need to confirm this with Motr team.

Reply:
Should we keep it then? or you want to confirm it with Motr before megre?

Reply:
Taking confirmation with Motr might take time and I don't think this is blocking change. So I feel even if we merge with this nothing will break.